.github/ISSUE_TEMPLATE/new-question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Question
+about: Ask a question about our implementation of LocalEGA
+
+---
+
+### Description
+<!-- Your question goes here. Keep it as much straightforward as you can -->
+
+### Related issues
+<!-- Related issues to this question go here -->
+
+### Mentions
+<!-- Should you want to direct the question to any of our developers, do it here -->
```

Comment:
I don't think this section is needed.

Reply:
Depends on the kind of question, right? It might be related to some specific part of the code, so therefore targeted to an individual.